spec/features/availability_spec.rb

```diff
@@ -37,21 +88,8 @@
     end
```

Comment:
Can you update test `An item reserved for controlled digital lending' ` with a record that is on CDL and not deactivate it?  ?

Reply:
We're skipping these because they hit live data that keeps changing. These are not tested in CI and will all eventually fail. They are not failing as a result of our PR. The idea is that we can eventually move the tests for this functionality over to Jest as we build out more of the Alma functionality.